package.json

```diff
@@ -25,6 +25,7 @@
     "build:web": "pnpm prebuild && pnpm turbo build --filter=./packages/web/* --color",
     "cargo:fix": "bash ./scripts/fix-symbolic-link.sh",
     "docker:build": "docker buildx bake -f docker/docker.hcl --load build",
+    "docker:override-up": "docker compose -f ./docker/docker-compose.override.yml up -d --remove-orphans ",
```

Comment:
what is this used for? Why do you need to customize your local setup?

Reply:
I have stuff running on those ports. I have local postgres and redis for example. If this is a blocker I can remove and create just local alias for this command just in my `.zshrc`